src/var/mash.jl

```diff
@@ -0,0 +1,57 @@
+# MinHash
+# ========
+#
+# Functions to get MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
```

Comment:
Coudn't this be obtained from Distances.jl? Then it could be `jaccarddistance(s1, s2) = Distances.jaccard(s1.sketch, s2.sketch)` or whatever